ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/winc/provision/cucushift-installer-rehearse-aws-ipi-winc-provision-chain.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/winc/provision/cucushift-installer-rehearse-aws-ipi-winc-provision-chain.yaml",
```

Comment:
this ocp is ovn, so we need mv to `ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/winc`

Reply:
yes